medusa-example.ini

```diff
@@ -23,6 +23,7 @@
 ;nodetool_password_file_path = <path to nodetool password file>
 ;nodetool_host = <host name or IP to use for nodetool>
 ;nodetool_port = <port number to use for nodetool>
+;nodetool_version_cmd = nodetool version
```

Comment:
This can be removed now.

Reply:
👍🏻

Reply:
Rebased w/ squash, sorry about that.  Ready for a re-check.

Reply:
The line is still there and now you have some extra commits that don't belong to this PR :)